security/advancedtls/go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/google/go-cmp v0.6.0
 	golang.org/x/crypto v0.33.0
 	google.golang.org/grpc v1.70.0
-	google.golang.org/grpc/examples v0.0.0-00010101000000-000000000000
+	google.golang.org/grpc/examples v0.0.0-20230224211313-3775f633ce20
```

Comment:
Why does the advancedtls module depend on examples?

Reply:
The integration test uses protos from the example

https://github.com/grpc/grpc-go/blob/5ac904279519840b8bd2f620be54d3d86d43e695/security/advancedtls/advancedtls_integration_test.go#L36